stdlib/sqlite3/dbapi2.pyi

```diff
@@ -420,15 +419,15 @@ class PrepareProtocol:
 
 class ProgrammingError(DatabaseError): ...
 
-class Row(Generic[_T_co]):
-    def __init__(self, __cursor: Cursor, __data: tuple[_T_co, ...]) -> None: ...
+class Row:
+    def __init__(self, __cursor: Cursor, __data: tuple[Any]) -> None: ...
```

Comment:
```suggestion
    def __init__(self, __cursor: Cursor, __data: tuple[Any, ...]) -> None: ...
```
A tuple containing any number of objects, not a tuple containing a single object.

Reply:
Oops...  thanks.